### DISCUSSION TITLE
Laravel / LDAPRecord, how to use ldap authentication on Login

### DISCUSSION THREAD

**Discussion by ThierryLesire** [DirectoryTree/LdapRecord#467] [General]
Hello all,
I'm new on Laravel and LDAP Record.
I ever use LDAP Record on classes and it work fine.
But I can't succeded to have first page login with LDAP Authentication
Last time I had "Authentication user provider [ldap] is not defined."
Maybe I done something wrong but I don't know what.
I used Laravel::UI for views and of course LDAPRecord.
Can I have a step by step implementation of this type of use?
DO we need "Users" table if it's only for authentication?

**Comment by stevebauman** [comment]
Hi @ThierryLesire,

There is a step-by-step guide on the LdapRecord.com documentation site. If you do not want a `users` database table, then you will need to follow the Plain Authentication guide here:

https://ldaprecord.com/docs/laravel/v2/auth/plain

**Comment by ThierryLesire** [comment]
Hi Steve,
it's that I done.
But I had errors... Maybe because used in Controllers... I don't know.
I ever tried successfully this method in classes or views.., but I try to implement in the Login Controller in Attempt method

**Reply by ThierryLesire** [reply]
It seems to be a config problem...
